tests/functional/builtins/codegen/test_abi_decode.py

```diff
@@ -421,15 +422,17 @@ def abi_decode(x: Bytes[160]) -> uint256:
 
 
 @pytest.mark.parametrize(
-    "output_typ1,output_typ2,input_",
+    "output_typ1,output_typ2,input_,error,error_property",
     [
-        ("DynArray[uint256, 3]", "uint256", b""),
-        ("DynArray[uint256, 3]", "uint256", b"\x01" * 128),
-        ("Bytes[5]", "address", b""),
-        ("Bytes[5]", "address", b"\x01" * 128),
+        ("DynArray[uint256, 3]", "uint256", b"", ExecutionReverted, ""),
+        ("DynArray[uint256, 3]", "uint256", b"\x01" * 128, EvmError, "OUT_OF_GAS_ERROR"),
+        ("Bytes[5]", "address", b"", ExecutionReverted, ""),
+        ("Bytes[5]", "address", b"\x01" * 128, EvmError, "OUT_OF_GAS_ERROR"),
```

Comment:
out of scope but these should really all be ExecutionReverted actually; it should be fixed by https://github.com/vyperlang/vyper/pull/3925